spec/factories/bundle_instance.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :bundle_instance do
+    bundle { FactoryGirl.create(:application_bundle).bundle }
```

Comment:
An `application_instance` ? Or do you mean `application_bundle` ? You can for sure drop `FactoryGirl`. But I would almost argue anywhere you need the application_bundle, you should
```ruby
application_bundle = create(:application_bundle)
bundle_instance = create(:bundle_instance, bundle: application_bundle.bundle)
```
But maybe that isn't the right way to do it and this is the right way.
My way means that application bundle isn't going to be created every time you create a bundle. Only when you explicitly create an application bundle because you need one, will it be created. Should be faster this way.